draft-ldbc-cats-framework.md

```diff
@@ -188,10 +188,13 @@ Computing-aware forwarding (or steeting, computing):
   :  A forwarding (or steeting, computing) scheme which takes as input a set of metrics that reflect the capabilities and state of computing resources.
 
 Service request:
- : The request for a specific service. Such a request is steered to a service contact instance.
+ : A request to access or invoke a specific service. Such a request is steered to a service contact instance via CATS-Forwarders.
+ : A service request is placed using service-specific protocols.
+ : Service requests are not explicitly sent by clients to CATS-Forwarders.
 
 CATS-Forwarder:
- : A network device that makes forwarding decisions based on CATS information to steer traffic specific to a  service request towards a corresponding yet selected service contact instance. The selection of a service contact instance relies upon a multi-metric CATS-based path computation. A CATS-Forwarder may behave as Ingress or Egress CATS-Forwarder.
+ : A network entity that makes forwarding decisions based on CATS information to steer traffic specific to a  service request towards a corresponding yet selected service contact instance. The selection of a service contact instance relies upon a multi-metric CATS-based path computation.
```

Comment:
Does CATS information contains network metric? What is CATS-based path computation

Reply:
We do have the following: 

"   Computing-Aware Traffic Steering (CATS):  A traffic engineering
      approach [I-D.ietf-teas-rfc3272bis] that takes into account the
      **dynamic nature of computing resources and network state** to
      optimize service-specific traffic forwarding towards a given
      service contact instance.  Various relevant metrics may be used to
      enforce such computing-aware traffic steering policies."

That's said, I think it is better if we simply remove "CATS-based" from the text you commented.